doc/content/source/materials/SteelCreepDamage.md

```diff
@@ -0,0 +1,42 @@
+# Steel Creep Damage
+
+!syntax description /Materials/SteelCreepDamage
+
+## Description
+
+The `SteelCreepDamage` can model the mechanical failure of steel due to creep [!cite](oh2011creep).
+
+Damage is computed by accumulating the creep strain rate normalized with the multi-axial creep ductility. The accumulated damage is 
+\begin{equation}
+\omega=\sum{\Delta\omega = \sum{\frac{\Delta\varepsilon_{x}}{\varepsilon^{*}_{f}}}}
+\end{equation}
+
+The triaxial creep ductility $\varepsilon^{*}_{f}$ can be obtained from the uniaxial creep ductility $\varepsilon_f$ using the following expression
+
+\begin{equation}
+\varepsilon^{*}_{f} = \varepsilon_{f} \frac{\sinh{\frac{2.0}{3.0} \frac{n - 0.5}{n + 0.5}}}{\sinh{2 \frac{n - 0.5}{n + 0.5} h}}
+\end{equation}
+
+where $n$ is the creep power law exponent, $h$ is the stress triaxiality.
+
+## Implementation and Usage
+
+The effects of damage can be applied all at once or progressively. In essence, this object allows the user to select an initial value of the damage index at which the load carrying capacity of the finite element will start to get decreased linearly (`initial_damage_value`). In addition, the user can select the factor that will reduce the load-carrying capacity (stress) through the input parameter `reduce_factor`. These two parameters should be chosen such that material damage consistently reduces material point's stresses. For example, for steel, `initial_damage_value` should be selected to be close to 1.0 so that the load-carrying capacity of the finite element is lost quickly. 
```

Comment:
When I look at the code, I'm not seeing how it gets reduced linearly. It looks like it just gets instantaneously divided by the factor. Could you make sure that this is consistent with what the code does?

Reply:
Here, `threshold` is the current value of damage. As it goes from `initial_damage_value` to `1.0`, the actual factor increments from (potentially) zero to the user-prescribed value of `_reduce_factor`. I do not think this has much of practical importance, just did it this way in case we needed to make the stress evolution numerically smoother in some cases.

```if (_use_old_damage)
    threshold = MetaPhysicL::raw_value(_damage_index_old[_qp]);
  else
    threshold = MetaPhysicL::raw_value(_damage_index[_qp]);

  if (threshold < _initial_damage_value)
    return;

  if (threshold > 1.0)
    threshold = 1;

  stress_new /= (threshold - _initial_damage_value) / (1 - _initial_damage_value) * _reduce_factor;
}